src/components/credentials/credentialsEmptyState.js

```diff
@@ -8,11 +8,11 @@ const CredentialsEmptyState = ({ onAddCredential, onAddSource }) => (
     <Row>
       <EmptyState className="full-page-blank-slate">
         <EmptyState.Icon />
-        <EmptyState.Title>Welcome to {helpers.RH_BRAND && 'Red Hat'} Entitlements Reporting</EmptyState.Title>
+        <EmptyState.Title>Welcome to {helpers.RH_BRAND && 'Red Hat'} discovery</EmptyState.Title>
```

Comment:
Causes an extra space `Welcome to__discovery` when there is no RH_BRAND.

Reply:
indeed, it just happens to take advantage of the older html and browser behavior where double spaces display as a single, plus the screen reader seems to skip it.

i suppose it could also be formatted as `{helpers.RH_BRAND && 'Red Hat '}discovery `

Reply:
went ahead and updated to remove the spacing